README.md

```diff
@@ -45,6 +34,20 @@ to developers/contributors.
 - [X] Implement Crypto
 - [X] Create Messenger
 
+## Q&A:
```

Comment:
@sudden6 I agree with the idea of it being needed. But I don't think it belongs in the main readme. IMO, the whole documentation can/should be rewritten and reorganized. But that's out of the scope of this pull.